packages/craco-less/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@semantic-ui-react/craco-less",
-  "version": "2.0.2",
+  "version": "2.1.0",
```

```suggestion
  "version": "3.0.0",
```

Let's bump this to v3 as it's a major bump of core dependency